### ISSUE TITLE
ME interface rendering error

### ISSUE THREAD

**Issue by zeng-github01** [AE2-UEL/NAE2#61]
When using Pattern Multi-Tool, after opening the interface, the interactive UI of the tool will be rendered on the left side of the GUI. This will cause the buttons on the left side of the interface GUI to be blocked.

**Comment by NotMyWing** [comment]
Known backburner problem, will have to do something about it sooner or later.

**Comment by zeng-github01** [comment]
@NotMyWing Very severe misalignment after using recent AE2-UEL action build

![image](https://github.com/AE2-UEL/NAE2/assets/95841646/d19c0013-950f-4375-90a0-f826e1210104)

**Comment by NotMyWing** [comment]
Yeah, I'm aware of this. Will be fixed once AE2-UEL is published to CurseForge.
